test/k8sT/Updates.go

```diff
@@ -232,6 +233,24 @@ func InstallAndValidateCiliumUpgrades(kubectl *helpers.Kubectl, oldHelmChartVers
 
 	testfunc := func() {
 		By("Deleting Cilium and CoreDNS...")
+
+		// Create a persistent connection to test that it persists
+		// across cilium reloads.
+		var persistenConnEC atomic.Int64
```

Comment:
The zero value of this atomic int is `0`. Later we assert that it is equal to `0`. How do we know if it is asserting against actually getting a `0`, or we happened to never write to the atomic variable and because it's zero-valued, we get a `0`?

Reply:
So there's a more interesting issue at play that I need to solve and that is the issue of persistent connections. I'm realizing that netcat can't actually do that because it's half-duplex so it won't close without an explicit "EOF". On some level I don't care if it exits "0". The issue is if a persistent connection gets destroyed by a restart.